### ISSUE TITLE
[JS-Controller 3.x] Object control.fan_power is invalid: obj.type has to exist

### ISSUE THREAD

**Issue by DutchmanNL** [iobroker-community-adapters/ioBroker.mihome-vacuum#114]
This object will not be created in future versions. Please report this to the developer.

```

undefined2020-04-08 20:43:27.389 - info: host.ioBroker "system.adapter.mihome-vacuum.0" enabled
--
2020-04-08 20:43:27.406 - info: host.ioBroker instance system.adapter.mihome-vacuum.0 started with pid 8752
2020-04-08 20:43:28.067 - warn: mihome-vacuum.0 (8752) Object mihome-vacuum.0.info.dnd is invalid: obj.common.type has an invalid value (switch) but has to be one of number, string, boolean, array, object, mixed, file, json
2020-04-08 20:43:28.067 - warn: mihome-vacuum.0 (8752) This object will not be created in future versions. Please report this to the developer.
2020-04-08 20:43:28.203 - info: mihome-vacuum.0 (8752) starting. Version 2.0.7 in /opt/iobroker/node_modules/iobroker.mihome-vacuum, node: v10.19.0, js-controller: 3.0.3
2020-04-08 20:43:28.229 - info: mihome-vacuum.0 (8752) Expert mode enabled, states created
2020-04-08 20:43:28.229 - info: mihome-vacuum.0 (8752) Create state clean_home for controlling by cloud adapter
2020-04-08 20:43:28.235 - info: mihome-vacuum.0 (8752) Disconnect
2020-04-08 20:43:28.254 - info: mihome-vacuum.0 (8752) change states from State control.fan_power
2020-04-08 20:43:28.254 - info: mihome-vacuum.0 (8752) extend state mop for State control.fan_power
2020-04-08 20:43:28.255 - info: mihome-vacuum.0 (8752) New generation or new fw detected, create new states goto and zoneclean
2020-04-08 20:43:28.261 - info: mihome-vacuum.0 (8752) Connected
2020-04-08 20:43:28.273 - info: mihome-vacuum.0 (8752) create states for water box
2020-04-08 20:43:28.274 - info: mihome-vacuum.0 (8752) create states for water box filter
2020-04-08 20:43:28.757 - info: mihome-vacuum.0 (8752) set next timer: not available
2020-04-08 20:43:30.254 - warn: mihome-vacuum.0 (8752) Object control.fan_power is invalid: obj.type has to exist
2020-04-08 20:43:30.255 - warn: mihome-vacuum.0 (8752) This object will not be created in future versions. Please report this to the developer.
2020-04-08 20:43:31.248 - info: mihome-vacuum.0 (8752) create state for carpet_mode


```

**Comment by mswiege** [comment]
Is this really still valid? 
The message about info.dnd was fixed in #116 and the control.fan_power object has a type defined (since years) (https://github.com/iobroker-community-adapters/ioBroker.mihome-vacuum/blame/master/io-package.json#L441) and I don't see this warning message in my logs.

**Comment by Apollon77** [comment]
no was a false positive by controller
